utils/mods.py

```diff
@@ -240,6 +254,25 @@ class RegistryMod(Enum):
         pattern_stubs=moreiotas_stubs,
     )
 
+    HexTweaks = HexTweaksRegistryModInfo(
+        name="HexTweaks",
+        directory="HexTweaks",
+        book=hextweaks_parse_book(
+            "HexTweaks/common/src/main/resources",
+            "common/src/main/java",
+            "HexMod/Common/src/resources",

```

Comment:
Fixes to make the paths not invalid:
```suggestion
            "HexTweaks/common/src/main/java",
            "HexMod/Common/src/main/resources",
```